tests/qa_automation/xfstests_device.pm

```diff
@@ -0,0 +1,115 @@
+# Copyright © 2012-2016 SUSE LLC
```

Comment:
I'm not good at Copyright, I just copy it from previous code. Is that better to change to 2017 or something?
# Copyright © 2012-2017 SUSE LLC